apps/typegpu-docs/src/content/docs/guides/bind-groups.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Bind Groups
+description: A guide on how to use the TypeGPU Bind Groups and Bind Group Layouts.
+draft: true
+---
+
+## What are Bind Groups and Bind Group Layouts?
+
+If you are already familiar with WebGPU, you might know what bind groups and bind group layouts are. In that case, you can skip this section.
```

Comment:
We can link the next section for convenience. This link should work, but please confirm 🤞.

```suggestion
If you are already familiar with WebGPU, you might know what bind groups and bind group layouts are. In that case, you can [#typegpu-approach](skip to the next section).
```

Reply:
`[skip to the next section](#typegpu-approach)` works 👍